lib/import/brca/providers/royal_marsden/royal_marsden_handler.rb

```diff
@@ -29,8 +29,8 @@ class RoyalMarsdenHandler < Import::Germline::ProviderHandler
           TEST_TYPE_MAP = { 'affected' => :diagnostic,
                             'unaffected' => :predictive }.freeze
           # rubocop:disable Lint/MixedRegexpCaptureTypes
-          CDNA_REGEX_PROT = /c\.(?<cdna>.+)(?=(?<separtors>_|;.)p\.(?<impact>.+))/i.freeze
-          CDNA_REGEX_NOPROT = /c\.(?<cdna>.+)/i.freeze
+          CDNA_REGEX_PROT = /c\.(?<cdna>.+)(?=(?<separtors>_|;.)p\.(?<impact>.+))/i
```

Comment:
is that supposed to be `separtors` ?

Reply:
@lauramccluskey1 should this be `separators`?

Reply:
See commit [a7f7fa0](https://github.com/NHSDigital/data_management_system/pull/67/commits/a7f7fa08076bae67d5b8bd6eedb4d9ba38bbe850)- tested and counts are still the same